setup.cfg

```diff
@@ -10,5 +10,4 @@ exclude =
   tests/.tests
 
 [tool:pytest]
-addopts = --flake8
 flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
```

Comment:
This line can go away as well (but leave the section heading to prevent pytest from scanning parent directories).